lib/rules/no-duplicate-selectors/__tests__/index.mjs

```diff
@@ -100,7 +103,16 @@ testRule({
 				{
 					message: messages.rejected('.a', 1),
 					line: 1,
-					column: 1,
+					column: 5,
+					endLine: 1,
+					endColumn: 7,
+				},
+				{
+					message: messages.rejected('.a', 1),
+					line: 1,
+					column: 9,
+					endLine: 1,
+					endColumn: 11,
 				},
```

Comment:
We now have two warnings instead of only one, relevant for : https://github.com/stylelint/stylelint/issues/7583